src/pyrenew/process/rtperiodicdiffar.py

```diff
@@ -179,17 +155,25 @@ def sample(
         n_periods = (duration + self.period_size - 1) // self.period_size
 
         # Running the process
-        ar_diff = FirstDifferenceARProcess(self.name, autoreg=b, noise_sd=s_r)
-        log_rt = ar_diff.sample(
-            duration=n_periods,
-            init_val=log_rt_rv[1],
-            init_rate_of_change=log_rt_rv[1] - log_rt_rv[0],
+        ar_diff = DifferencedProcess(
+            fundamental_process=ARProcess(name="first_diff_log_rt_ar"),
+            differencing_order=1,
+        )
+
+        log_rt = ar_diff(
+            n=n_periods,
+            init_vals=jnp.array([log_rt_rv[1]]),
+            autoreg=b,
+            noise_sd=s_r,
```

Comment:
`b` and `s_r` could have better names

Reply:
Out of scope for this PR imo, since this was an existing class.